### PR TITLE
[IMP] 16.0 Misc VoIP Version Specific Changes

### PR DESCRIPTION
**Pull request by tiku-odoo**
Version specific improvements for Odoo 16 pertaining to this PR: 

https://github.com/odoo/documentation/pull/5256

### CODE REVIEW

**Comment by tiku-odoo** [comment]
Hi @StraubCreative, 

This PR is ready for approval, it contains a few simple version-specific changes to the VOIP bundle for V16. 

Thanks! 
Tim :+1:

File: content/applications/general/voip/axivox/conference_calls.rst [hunk 1]
```diff
@@ -86,7 +86,7 @@ is then prompted to enter the :guilabel:`Access code` to enter the conference.
 Start call in Odoo
 ==================
 
-Anywhere in the Odoo database, open the *VoIP* widget, by clicking the :guilabel:`📞 (phone)` icon,
+Anywhere in the Odoo database, open the *VoIP* widget, by clicking the :guilabel:`☎️ (phone)` icon,
 located in the upper-right corner. Then, dial the specific extension number for the conference, and
 click the :guilabel:`📞 (phone)` icon.
```
Comment (StraubCreative) [inline content/applications/general/voip/axivox/conference_calls.rst:91]: Missed an emoji I think. Added some more context too so it didn't read as redundant.
```suggestion
Anywhere in the Odoo database, open the *VoIP* widget, by clicking the :guilabel:`☎️ (phone)` icon,
located in the upper-right corner. Then, dial the specific extension number for the conference, and
click the :guilabel:`☎️ (phone)` icon again, this time on the *VoIP* widget itself.
```

**Review by StraubCreative — APPROVED** [review]
Hi @tiku-odoo 
Clean PR. I caught one small thing which I'll fix and push up in a sec.
Thanks for the adjustments here!

File: content/applications/general/voip/axivox/conference_calls.rst (hunk 1 above)
Comment (StraubCreative) [reply content/applications/general/voip/axivox/conference_calls.rst:91]: Whoop. I corrected myself— what you had originally was accurate. Disregard please!

**Comment by StraubCreative** [comment]
@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo r-

**Comment by StraubCreative** [comment]
@fw-bot up to saas-16.4

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**